src/libraries/System.Formats.Cbor/src/System/Formats/Cbor/Reader/CborReader.Integer.cs

```diff
@@ -5,25 +5,19 @@
 
 namespace System.Formats.Cbor
 {
+    /// <summary>A stateful, forward-only reader for Concise Binary Object Representation (CBOR) encoded data.</summary>
     public partial class CborReader
     {
-        // Implements major type 0,1 decoding per https://tools.ietf.org/html/rfc7049#section-2.1
```

Comment:
Looks like the tool deleted a non-xml comment here.

Reply:
This shouldn't happen with the latest bits. Did you get a chance to update the tool? I fixed this last week.

Reply:
So, it's not missing, it's been placed _under_ the XML docs in line 20: https://github.com/dotnet/runtime/pull/49506/files/95ce9f2f792b6f4ecded4b45c7bd3eb337ee04e8#diff-940ced5667ba15494ec71aeeb271c77551a18000f6cc3c0df17529cbf29d7e68R20

The problem however is that the comment is not specific to the method, it's class wide.